libcodechecker/cmd/cmd_line_client.py

```diff
@@ -321,11 +318,7 @@ def handle_list_results(args):
                 rows.append(
                     (checked_file, res.checkerId, sev, res.checkerMsg))
 
-        if args.output_format == 'csv':
-            writer = csv.writer(sys.stdout)
-            writer.writerows(rows)
-        else:
-            print_table(rows)
+        print(twodim_to_str(args.output_format, header, rows))
```

Comment:
Yes, by accidentally making a typo like `jsson` it will be not a valid format and the exception will go back to the user, or is it handled somewhere else and I missed it?